### PR TITLE
(chore) Change slot name to launch billing on Visit form

### PR DESCRIPTION
**Pull request by ODORA0**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary
In the patient chart we added a generic slot to mount anything in the start visit form, this PR renames the slot to the generic slot to mount billing within the start visit form

## Screenshots
image

### CODE REVIEW

File: src/routes.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     },
     {
       "name": "billing-checkin-form",
-      "slot": "billing-checkin-slot",
+      "slot": "extra-visit-attribute-slot",
```
Comment (CynthiaKamau) [inline src/routes.json:51]: I think adding the word billing in there would be more intuitive eg `billing-visit-attribute-slot` or `extra-billing-visit-attributes-slot`

Comment (pirupius) [reply src/routes.json:51]: The intention was to have a generic slot name that can be reused by other implementations and anyone who would like to add any extra visit attributes. This slot is defined within the patient chart in the start visit form.

**Review by pirupius — COMMENTED** [review]
@ODORA0 this PR seems to depend on the one from the community. So am guessing this should wait for that so they don'r conflict

File: src/routes.json (hunk 1 above)
Comment (CynthiaKamau) [reply src/routes.json:51]: In the start visit form, there is already the ability to configure visit attributes, in this case, we are only interested in the billing ones : https://github.com/openmrs/openmrs-esm-patient-chart/blob/ff152cfaa77b5141d1387bec922d3417a00d72b3/packages/esm-patient-chart-app/src/config-schema.ts#L37

**Pull request merged**